printing_element_of_enum.c

```diff
@@ -0,0 +1,11 @@
+#include<stdio.h>
+enum year{Jan=1, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct=20, Nov, Dec};
+void main()
+{
```

Comment:
Print value of Jan and Dec before the loop execution.

Reply:
Sir,I have done the required changes.